react/react1/week1/react-week1/src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App.js';
```

Comment:
Instead of using "App" as filename  i would recommend you to change it to "ToDo" so that the filename reflects the component you are exporting from that file. That will improve readability.